packages/toolchain/elemental-cli/collection.yaml

```diff
@@ -11,7 +11,7 @@ packages:
       autobump.revdeps: "true"
       autobump.strategy: "git_hash"
       autobump.git.branch: "main"
-      git.hash: "4ca13d5eff80257dbe9b31e91ac24352b9118b5f"
+      git.hash: "2a84274944eb74c01bd6a9a86a3c4712aaedc1a2"
```

Comment:
```suggestion
      git.hash: "4f8bf31dff1e5b539a6654810317c851faf70732"
```

Reply:
Thank you @mudler I was wondering why it is failing, rebased